brave/brave-core
# fix shields and rewards not being translated to Norwegian

fix https://github.com/brave/brave-browser/issues/3425

### Test Plan

1. Change your OS locale to Norwegian
2. Ensure both Shields and Rewards extensions are translated
